app/code/Magento/InventoryCatalog/Api/DefaultSourceRepositoryInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\InventoryCatalog\Api;
+
+use Magento\InventoryApi\Api\Data\SourceInterface;
+
+/**
+ * Represents default source
+ *
+ * @api
+ */
+interface DefaultSourceRepositoryInterface
+{
+    const DEFAULT_SOURCE= 1;
```

Comment:
currently, we have a requirement to provide a description to all the constants declared,
because constant introduces public contract the same as the public method.

Reply:
But in this case, I propose to remove this constant and leave an interface method 
```
DefaultSourceResolverInterface::getId(): int
```
in this case, somebody could customize it. For example, change ID of the default source/stock

Having constant - there is no such possibility